frontend/src/css/codemirror.css

```diff
@@ -30,7 +26,6 @@
 /* -- Tooltips: code completion and type hints -- */
 
 .cm .cm-tooltip {
-  font-family: var(--monospace-font);
```

Comment:
i find this MUCH easier to read and adds more content to the same area of tooltip

Reply:
nice, i do think it's easier to read. only thing is that the prose and code look to be different sizes (not sure if they actually are), with the code looking bigger — possible to shrink the code size or increase the prose size?

Reply:
I'm a little split actually on which font style I prefer

Reply:
if you are split, i think we should get rid of it. i think it looks much better and similar to existing IDEs. but there is a bug in the python docstring -> markdown in that it creates some extra new lines. ill revert for now